docs/nodejs-runtime/master.adoc

```diff
@@ -17,6 +17,12 @@ include::topics/runtime-info-community-version.adoc[leveloffset=+1]
 
 include::topics/dev-guide-mission-intro.adoc[leveloffset=+1]
 
+
+[[nodejs-available-missions-and-boosters]]
```

Comment:
Please use the `[#heading]` notation instead of `[[heading]]`.

Valid everywhere (can be fixed using a regular expression in one go, so please don't change every occurrence manually. I can help with the mass change if needed).

Reply:
@tradej - will touch base to do the mass change for heading notation.